model/licensing.py

```diff
@@ -1293,7 +1293,7 @@ class DeliveryMechanism(Base, HasFullTableCache):
     OVERDRIVE_DRM = u"Overdrive DRM"
     BEARER_TOKEN = u"application/vnd.librarysimplified.bearer-token+json"
 
-    STREAMING_PROFILE = ";profile=http://librarysimplified.org/terms/profiles/streaming-media"
+    STREAMING_PROFILE = ';profile="http://librarysimplified.org/terms/profiles/streaming-media"'
```

Comment:
We have tests ready to go:

https://github.com/NYPL-Simplified/android/blob/0cab69fd2a4e2e44d49a6115ee3413f1e969d274/simplified-tests/src/main/java/org/nypl/simplified/tests/mime/MIMEParserContract.kt#L53